starlark-repl/Cargo.toml

```
@@ -34,3 +34,7 @@ tempfile = ">=3, <3.0.5"
 
 [[bin]]
 name = "starlark-repl"
+
+[features]
+default =["linked_hash_set"]
```

nit: missing whitespace.

```suggestion
default = ["linked_hash_set"]
```